src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml.cs

```diff
@@ -243,6 +243,9 @@ private void OpenFileLocation(RecentItem item)
 
 		private void OpenProperties(RecentItem item)
 		{
+			if (item is null)
+				return;
+
```

Comment:
This check is also good, but it is more likely that the `flyout` is null.

Reply:
Thanks